apps/src/aiComponentLibrary/suggestedPrompt/suggested-prompt.module.scss

```diff
@@ -1,38 +1,61 @@
-@import "color.scss";
-@import "font.scss";
-@import "@code-dot-org/component-library-styles/typography.module";
+@use '@code-dot-org/component-library-styles/mixins';
+@import 'color.scss';
```

Comment:
If you replace the focus styles with the mixin [here](https://github.com/code-dot-org/code-dot-org/pull/69894/files#r2590772247), this can be removed.